plugin/storage/badger/config.go

```diff
@@ -14,14 +14,8 @@ import (
 	"go.uber.org/zap"
```

Comment:
I would not rename the file but split it - the Config and DefaultConfig and Validate functions (basically things that are relevant for v2) can go to config.go, and the CLI flags logic should stay in options.go